### PR TITLE
fix(doc): improve documentation install using ISO

### PR DESCRIPTION
**Pull request by lpinsivy**
ref: #5772

### CODE REVIEW

File: doc/en/installation/from_centreon.rst [hunk 1]
```diff
@@ -219,19 +219,19 @@ To start the monitoring engine :
  3.	Uncheck **Generate Configuration Files** and **Run monitoring engine debug (-v)**
  4.	Check **Move Export Files** and **Restart Monitoring Engine**
  5.	Click on **Export** again
- 6.   Log into the ‘root’ user on your server
```
Comment (kduret) [inline doc/en/installation/from_centreon.rst]: Indentation does not seem to be the same than previous points

File: doc/fr/installation/from_centreon.rst [hunk 2]
```diff
@@ -233,19 +233,19 @@ Pour démarrer l'ordonnanceur de supervision :
  3.	Décochez **Générer les fichiers de configuration** et **Lancer le débogage du moteur de supervision (-v)**
  4.	Cochez **Déplacer les fichiers générés** ainsi que **Redémarrer l'ordonnanceur**
  5.	Cliquez à nouveau sur **Exporter**
- 6.   Connectez-vous avec l'utilisateur 'root' sur votre serveur
+ 6. Connectez-vous avec l'utilisateur 'root' sur votre serveur
```
Comment (kduret) [inline doc/fr/installation/from_centreon.rst]: Indentation does not seem to be the same than previous points

**Comment by lpinsivy** [comment]
done for indentation

**Comment by adr-mo** [comment]
Steps are well formated.

Looks good to me

**Pull request merged**